infrastructure/stacks/development-pipeline/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
```

Comment:
This file can be removed and the providers.tf can be updated to the code below. This means that you don't have to tag every resource
```
provider "aws" {
  profile = var.aws_profile
  region  = var.aws_region

  default_tags {
    tags = {
      "Programme"   = "uec"
      "Service"     = "uec-dos-int"
      "Product"     = "uec-dos-int"
      "Environment" = var.profile
    }
  }
}
```